### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by qpetraroia**

### CODE REVIEW

**Review by sabbour — APPROVED** [review]
Looks good.

**Review by davidgamero — APPROVED** [review]
Lgtm

File: README.md [hunk 1]
```diff
@@ -38,15 +37,11 @@ To properly use Draft to containerize your application, create manifest files, a
...
-**Draft Set Up GitHub OpenID Connect** automates the GitHub OIDC setup process for your project.
-
-**Draft Generate Workflow** generates a GitHub Actions workflow for automatic build and deploy to a Kubernetes cluster.
-
-**Draft Add Web App Routing Annotations** annotates your Kubernetes service manifest to expose it to the internet through Web App Routing.
-
 ## Telemetry
 
-This extension collects telemetry data to help us build a better experience for building applications with Azure Kubernetes Service and VS Code.
+This extension collects telemetry data to help us build a better experience for building applications with Azure Kubernetes Service and VS Code. We only collect the following data:
```
Comment (OliverMKing) [inline README.md:42]: This is incorrect. We rely on the official `VS Code Azure SDK` tooling. It automatically tracks a few [other fields](https://github.com/Microsoft/vscode-azuretools/blob/main/utils/README.md#telemetry-and-error-handling).

**Review by OliverMKing — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -38,15 +37,11 @@ To properly use Draft to containerize your application, create manifest files, a
```
Comment (OliverMKing) [inline README.md:37]: The list of commands is outdated 

`Draft Create Deployment Files` was split into `Draft a Dockerfile` and `Draft a Kubernetes Deployment and Service`

**Review by OliverMKing — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (qpetraroia) [reply README.md]: @OliverMKing added changes, want to take a look?

**Review by OliverMKing — APPROVED** [review]

**Pull request merged**